exports/@material-ui/core/4.10.0.json

```diff
@@ -0,0 +1,116 @@
+  "./InputLabel": "./esm/InputLabel/InputLabel.js",
+  "./LinearProgress": "./esm/LinearProgress/LinearProgress.js",
+  "./Link": "./esm/Link/Link.js",
+  "./List": "./esm/List/List.js",
+  "./ListItem": "./esm/ListItem/ListItem.js",
+  "./ListItemAvatar": "./esm/ListItemAvatar/ListItemAvatar.js",
+  "./ListItemIcon": "./esm/ListItemIcon/ListItemIcon.js",
+  "./ListItemSecondaryAction": "./esm/ListItemSecondaryAction/ListItemSecondaryAction.js",
+  "./ListItemText": "./esm/ListItemText/ListItemText.js",
+  "./ListSubheader": "./esm/ListSubheader/ListSubheader.js",
+  "./Menu": "./esm/Menu/Menu.js",
+  "./MenuItem": "./esm/MenuItem/MenuItem.js",
+  "./MenuList": "./esm/MenuList/MenuList.js",
+  "./MobileStepper": "./esm/MobileStepper/MobileStepper.js",
+  "./Modal": "./esm/Modal/Modal.js",
+  "./NativeSelect": "./esm/NativeSelect/NativeSelect.js",
+  "./NoSsr": "./esm/NoSsr/NoSsr.js",
+  "./OutlinedInput": "./esm/OutlinedInput/OutlinedInput.js",
+  "./Paper": "./esm/Paper/Paper.js",
+  "./Popover": "./esm/Popover/Popover.js",
+  "./Popper": "./esm/Popper/Popper.js",
+  "./Portal": "./esm/Portal/Portal.js",
+  "./Radio": "./esm/Radio/Radio.js",
+  "./RadioGroup": "./esm/RadioGroup/RadioGroup.js",
+  "./RootRef": "./esm/RootRef/RootRef.js",
+  "./ScopedCssBaseline": "./esm/ScopedCssBaseline/ScopedCssBaseline.js",
+  "./Select": "./esm/Select/Select.js",
+  "./Slide": "./esm/Slide/Slide.js",
+  "./Slider": "./esm/Slider/Slider.js",
+  "./Snackbar": "./esm/Snackbar/Snackbar.js",
+  "./SnackbarContent": "./esm/SnackbarContent/SnackbarContent.js",
+  "./Step": "./esm/Step/Step.js",
+  "./StepButton": "./esm/StepButton/StepButton.js",
+  "./StepConnector": "./esm/StepConnector/StepConnector.js",
+  "./StepContent": "./esm/StepContent/StepContent.js",
+  "./StepIcon": "./esm/StepIcon/StepIcon.js",
+  "./StepLabel": "./esm/StepLabel/StepLabel.js",
+  "./Stepper": "./esm/Stepper/Stepper.js",
+  "./SvgIcon": "./esm/SvgIcon/SvgIcon.js",
+  "./SwipeableDrawer": "./esm/SwipeableDrawer/SwipeableDrawer.js",
+  "./Switch": "./esm/Switch/Switch.js",
+  "./Tab": "./esm/Tab/Tab.js",
+  "./TabScrollButton": "./esm/TabScrollButton/TabScrollButton.js",
+  "./Table": "./esm/Table/Table.js",
+  "./TableBody": "./esm/TableBody/TableBody.js",
+  "./TableCell": "./esm/TableCell/TableCell.js",
+  "./TableContainer": "./esm/TableContainer/TableContainer.js",
+  "./TableFooter": "./esm/TableFooter/TableFooter.js",
+  "./TableHead": "./esm/TableHead/TableHead.js",
+  "./TablePagination": "./esm/TablePagination/TablePagination.js",
+  "./TableRow": "./esm/TableRow/TableRow.js",
+  "./TableSortLabel": "./esm/TableSortLabel/TableSortLabel.js",
+  "./Tabs": "./esm/Tabs/Tabs.js",
+  "./TextField": "./esm/TextField/TextField.js",
+  "./TextareaAutosize": "./esm/TextareaAutosize/TextareaAutosize.js",
+  "./Toolbar": "./esm/Toolbar/Toolbar.js",
+  "./Tooltip": "./esm/Tooltip/Tooltip.js",
+  "./Typography": "./esm/Typography/Typography.js",
+  "./Zoom": "./esm/Zoom/Zoom.js"
+}
```

Comment:
Believe it or not, these are actually different across versions 🙃  so they can’t be generated